v2/notifier_test.go

```diff
@@ -187,6 +249,7 @@ func assertStackframesMatch(t *testing.T, expected []errors.StackFrame) {
 			if strings.HasSuffix(file, expectedFrame.File) && expectedFrame.Name == method {
 				lastmatch = index
 				matched++
+				break
```

Comment:
Can you also provide rationale for this change in the PR description? Why did it pass before and not now?

Reply:
It passed before because the stack was:
1. func.1
2. func
and each function got matched only once so `matched` count was correct.

When I added expansion of inlines the stack became:
1. func.1 (for go1.11 and crash for everything else)
2. func.1
3. func
that caused incorrect matching of actual and expected stacks - each actual stackframe could be matched many times - we did not stop comparing after finding first match.
Matched count exceeded expected stack length and panic happened on the next log. (out of bounds idx)
Notice that error_test.go has the break in the same place while matching stacks.